SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.UWP/Microsoft.AppCenter.Analytics.UWP.csproj

```diff
@@ -1,141 +1,29 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{200073BF-FA7E-42AF-8ED5-AA50BE7C0295}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Microsoft.AppCenter.Analytics</RootNamespace>
+    <TargetFrameworks>uap10.0.16299;net461</TargetFrameworks>
+    <Company>Microsoft Corporation</Company>
+    <Copyright>Microsoft Corp. All rights reserved.</Copyright>
+    <Version>3.0.1-SNAPSHOT</Version>
```

Comment:
Does start-version.sh picks it up?

Reply:
it should, we already have this type of projects. I'll check explicitly

Reply:
It works